### ISSUE TITLE
Error Add public sourse

### ISSUE THREAD

**Issue by esmelnikov** [StamusNetworks/scirius#156]
When I try to add the source "Abuse.ch SSL Blacklist" through the interface "Add public sourse" I get the following list of errors: (Scirius CE v3.0.0.)

![2018-10-11_091000](https://user-images.githubusercontent.com/26947034/46780353-b943f700-cd35-11e8-9b32-e8a8221c604a.png)

**Comment by pevma** [comment]
Seems your Suricata installation is an older version - needs to updated. Also look at ``/etc/scirius/local_settings.py`` to see if you have the correct path for the unix socket. (10x @regit )

**Comment by esmelnikov** [comment]
Path in /etc/scirius/local_settings.py correct/
Suricata version 4.1.0-dev (rev cba41207)

**Comment by pevma** [comment]
Could you share the path please as it is in the config ?

**Comment by esmelnikov** [comment]
SURICATA_UNIX_SOCKET = "/var/run/suricata/suricata-command.socket"

**Comment by pevma** [comment]
Yes - i can confirm. It seems the rules would need to be updated to use the latest suri dev tls keywords. Will investigate further.
